frontend/src/lib/lemon-ui/LemonInput/LemonInput.scss

```diff
@@ -97,6 +97,12 @@
     &.LemonInput--type-search {
         // NOTE Design: Search inputs are given a specific small width
         max-width: 240px;
+        border-radius: 0;
+
+        // Add a focus ring to the element (not the input) when focused
+        &:has(:focus-visible) {
+            outline: -webkit-focus-ring-color auto 1px;
```

Comment:
I see this on the search bar even after _clicking_ "Search", which is pretty ugly:
![Screenshot 2024-12-18 at 16 51 12](https://github.com/user-attachments/assets/4700a3d8-a77d-474e-be7a-9928c0bfb3b3)

We shouldn't _really_ need this overall, since there's active styling on inputs already. But if we do want to make focus styling more prominent on keyboard navigation, we should probably apply the same logic to all inputs, not just search ones (type="search" is used in many places)

Reply:
After the [recent PR](https://github.com/PostHog/posthog/pull/26992), I perhaps got ahead of myself. I think you're right though, so I have removed it (for now). I WILL add focus styles in the future though ;) super important for a11y. 

With that said, I want to make a setting screen for A11y, where users can turn this on/off. Other a11y features to toggle which come to mind: underline on links, high contrast mode, etc.